pretty() options required in types

Hey folks, I'm following the readme example

```typescript
const pino = require('pino')
const pretty = require('pino-pretty')
const logger = pino(pretty())
```

but I'm hitting the following type-check issue

```
src/index.ts:14:14 - error TS2554: Expected 1 arguments, but got 0.

14 app.use(pino(pretty()));
                ~~~~~~~~

  node_modules/pino-pretty/index.d.ts:16:29
    16 declare function PinoPretty(options: PrettyOptions_): PinoPretty.PrettyStream;
                                   ~~~~~~~~~~~~~~~~~~~~~~~
    An argument for 'options' was not provided.


Found 1 error in src/index.ts:14
```

which looks like from here where the options are required

https://github.com/pinojs/pino-pretty/blob/077216cb7287ff9f52980de8c61c44085a7112b5/index.d.ts#L16

I think the options argument should be optionally typed, otherwise typescript complains. Am I missing something, or is this an oversight?

As a workaround for now, this works

```typescript
app.use(pino(pretty({})));
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -13,7 +13,7 @@ import { DestinationStream } from 'pino';
 
 type LogDescriptor = Record<string, unknown>;
 
-declare function PinoPretty(options: PrettyOptions_): PinoPretty.PrettyStream;
+declare function PinoPretty(options?: PrettyOptions_): PinoPretty.PrettyStream;
 
 interface PrettyOptions_ {
   /**

--- a/test/types/pino-pretty.test-d.ts
+++ b/test/types/pino-pretty.test-d.ts
@@ -60,6 +60,7 @@ const options2: PrettyOptions = {
   mkdir: true,
 };
 
+expectType<PrettyStream>(pretty()); // #326
 expectType<PrettyStream>(pretty(options));
 expectType<PrettyStream>(PinoPrettyNamed(options));
 expectType<PrettyStream>(PinoPrettyDefault(options));

